config/database.yml

```diff
@@ -1,57 +1,26 @@
-# PostgreSQL. Versions 9.1 and up are supported.
+# MySQL. Versions 5.1.10 and up are supported.
 #
-# Install the pg driver:
-#   gem install pg
-# On OS X with Homebrew:
-#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
-# On OS X with MacPorts:
-#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
-# On Windows:
-#   gem install pg
-#       Choose the win32 build.
-#       Install PostgreSQL and put its /bin directory on your path.
+# Install the MySQL driver
+#   gem install mysql2
 #
-# Configure Using Gemfile
-# gem 'pg'
+# Ensure the MySQL gem is defined in your Gemfile
+#   gem 'mysql2'
+#
+# And be sure to use new-style password hashing:
+#   http://dev.mysql.com/doc/refman/5.7/en/old-client.html
 #
 default: &default
-  adapter: postgresql
-  encoding: unicode
-  # For details on connection pooling, see Rails configuration guide
-  # http://guides.rubyonrails.org/configuring.html#database-pooling
+  adapter: mysql2
```

Comment:
Won't this change of the production database adapter break the current deployment on Heroku, which is still using PostgreSQL?

I wonder if it might be better to switch to using the a DATABASE_URL environment variable, as the comments below in `database.yml` suggest...

Reply:
I believe Heroku is already using the `DATABASE_URL` environment variable, so it _should_ ignore this configuration file as the `DATABASE_URL` takes precedence (I think, can't find docs backing this up).